Sources/HTTP/HTTPResponse.swift

```diff
@@ -49,224 +49,159 @@ public enum HTTPTransferEncoding {
+    public let reasonPhrase: String
 
-    public init(rawValue: UInt16) {
-        switch rawValue {
-            case 100: self = .continue
-            case 101: self = .switchingProtocols
-            case 102: self = .processing
-            case 200: self = .ok
-            case 201: self = .created
-            case 202: self = .accepted
-            case 203: self = .nonAuthoritativeInformation
-            case 204: self = .noContent
-            case 205: self = .resetContent
-            case 206: self = .partialContent
-            case 207: self = .multiStatus
-            case 208: self = .alreadyReported
-            case 226: self = .imUsed
-            case 300: self = .multipleChoices
-            case 301: self = .movedPermanently
-            case 302: self = .found
-            case 303: self = .seeOther
-            case 304: self = .notModified
-            case 305: self = .useProxy
-            case 307: self = .temporaryRedirect
-            case 308: self = .permanentRedirect
-            case 400: self = .badRequest
-            case 401: self = .unauthorized
-            case 402: self = .paymentRequired
-            case 403: self = .forbidden
-            case 404: self = .notFound
-            case 405: self = .methodNotAllowed
-            case 406: self = .notAcceptable
-            case 407: self = .proxyAuthenticationRequired
-            case 408: self = .requestTimeout
-            case 409: self = .conflict
-            case 410: self = .gone
-            case 411: self = .lengthRequired
-            case 412: self = .preconditionFailed
-            case 413: self = .payloadTooLarge
-            case 414: self = .uriTooLong
-            case 415: self = .unsupportedMediaType
-            case 416: self = .rangeNotSatisfiable
-            case 417: self = .expectationFailed
-            case 421: self = .misdirectedRequest
-            case 422: self = .unprocessableEntity
-            case 423: self = .locked
-            case 424: self = .failedDependency
-            case 426: self = .upgradeRequired
-            case 428: self = .preconditionRequired
-            case 429: self = .tooManyRequests
-            case 431: self = .requestHeaderFieldsTooLarge
-            case 451: self = .unavailableForLegalReasons
-            case 500: self = .internalServerError
-            case 501: self = .notImplemented
-            case 502: self = .badGateway
-            case 503: self = .serviceUnavailable
-            case 504: self = .gatewayTimeout
-            case 505: self = .httpVersionNotSupported
-            case 506: self = .variantAlsoNegotiates
-            case 507: self = .insufficientStorage
-            case 508: self = .loopDetected
-            case 510: self = .notExtended
-            case 511: self = .networkAuthenticationRequired
-            default:  self = .other(statusCode: rawValue, reasonPhrase: "http_\(rawValue)")
-        }
+    public init(_ code: UInt16, _ reasonPhrase: String) {
```

Comment:
I have thought about this a little. The main reason why no argument labels are provided is HTTP status line. It has exactly the same parameters and ordering. `"200 OK"` translates to `HTTPResponseStatus(200, "OK")` very intuitively. With labels it will become more Swifty but less HTTP.